Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-resource/ProblemResource.tsx

```diff
@@ -21,35 +21,37 @@ const ProblemResource = ({ resource }: IProblemResourceProps) => {
     const {
         actions: {
             getProblemResourceFile,
```

Comment:
`get` here implies that this method returns a value. I guess, it does not return a file, but sets `problemResourceContent`. `load` is a better verb here.

Reply:
refactored code so that 'get' is relevant in the naming

Reply:
renamed to download